collections/concepts/what-is-the-proxy.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title:  "What Is the Proxy?"
+categories: concepts
+permalink: /concepts/what-is-the-proxy/
+nav_order: 4
+has_children: true
+has_toc: false
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+
+# What Is the Proxy?
+
+It is a common need to share data between software systems via HTTP based APIs, but what if the HTTP request requires a piece of sensitive data that you have tokenized and do not want to access directly within your application?
+
+The Proxy enables you to build HTTP requests containing sensitive tokenized data and send them from your systems in a secure and compliant manner without needing to access raw token data. 
+Your application can include token identifiers within the request payload and send this request through Basis Theory's systems to substitute token data into the request before forwarding it to the desired destination. 
+
+The end result is that you can call HTTP APIs with tokenized data without needing to touch the underlying sensitive data directly within your systems.
+
+
+## How It Works
+
+Your system initiates an outbound HTTP request to the Proxy hosted by Basis Theory.
+The request is transformed by detokenizing any standard token interpolation patterns included in the request (patterns of the form `{%raw%}{{tokenId}}{%endraw%}`, where `tokenId` is the id of a token created within your tenant). Any token interpolation patterns will be replaced with the detokenized `data` value of this token.
+Finally, the request is delivered to the destination URL that was specified through a `BT-PROXY-URL` HTTP header.
+
+![Outbound Proxy Diagram](/assets/images/what_is_the_proxy/outbound-proxy.png)
+
+The Proxy terminates the inbound TLS connection from your servers and initiates a new TLS connection to the destination in order to guarantee secure transmission of your sensitive token data.
+For this reason, we require the destination servers to support TLSv1.2+ and that the provided destination URL uses `https`.
+
+The application that is being used to call the Proxy must be explicitly granted `token:<classification>:use:proxy` permission on any tokens that are detokenized. 
+You should restrict which classifications of tokens an application has access to detokenize through the Proxy by only granting the minimum set of `token:<classification>:use:proxy` permissions that are necessary for your use case.
```

Comment:
Maybe link them to the place in docs where we explain classifications/permissions?